### ISSUE TITLE
Can you include Isar for comparison?

### ISSUE THREAD

**Issue by raphire08** [objectbox/objectbox-dart-performance#2]

**Comment by vaind** [comment]
There's already a contributor-provided PR for that: #1 but with some issues. If you'd like to contribute maybe try updating it and see if the issues are resolved?

**Comment by vaind** [comment]
FYI: we didn't want to do it before because we wanted to wait for Isar to mature a little more to be fair in the comparison.

**Comment by raphire08** [comment]
I checked the PR and there was class declaration error in generated file. I was able to remove the bug after the code was generated. But, I could not change the code to generate the correct file. I will look at it. Meanwhile, I ran the build in OnePlus5 and these were the results. Seems like Isar and ObjectBox both are comparatively faster than SqLite and Hive. But, as you said Isar is still in alpha so once can expect some issues while in production

<details>
 <summary>Results (screenshots)</summary>

![Screenshot_20210707-223913](https://user-images.githubusercontent.com/20361752/124864158-1435d600-dfd6-11eb-9327-f6e88de427b6.jpg)
![Screenshot_20210707-224312](https://user-images.githubusercontent.com/20361752/124864167-1730c680-dfd6-11eb-8797-9935dd7b838d.jpg)
![Screenshot_20210707-224325](https://user-images.githubusercontent.com/20361752/124864172-1861f380-dfd6-11eb-9529-b383a0204db0.jpg)
![Screenshot_20210707-224335](https://user-images.githubusercontent.com/20361752/124864174-18fa8a00-dfd6-11eb-9c4c-a39942477165.jpg)

</details>

-------
Edit by @vaind: fold screenshots to an expandable block so we don't have to scroll so much...
